.Documentation/articles/pro/03-desktop-mode.md

```diff
@@ -0,0 +1,62 @@
+
+### Moving around and rotating view
+
+The _novice_ desktop mode does not support moving around your training scene (locomotion) but to rotate your viewport around your current position.
+In order to rotate your viewport, simply click anywhere in your scene, keep the mouse button pressed and drag your mouse to any direction. 
+
+[![Looking around](../images/pro/03-mouse-viewport-control.gif "Looking around in the scene.")](../images/pro/03-mouse-viewport-control.gif)
+
+_Note:_ As a developer you can choose the mouse button in the _Mouse Controller_ of the _Desktop Mode Rig_ prefab (e.g. _[NOVICE_DESKTOP_MODE_RIG]_).
+
+[![Mouse Controller](../images/pro/03-mouse-controller.png "Mouse controller settings.")](../images/pro/03-mouse-controller.png)
+
+### Interacting with objects
+
+In _novice_ desktop mode, users interact via left mouse button. Moving the mouse pointer over interactable objects will highlight them to communicate a possible interaction with such objects.
+
+[![Selecting objects](../images/pro/03-mouse-over-highlighting.gif "Selecting objects.")](../images/pro/03-mouse-over-highlighting.gif)
+
+Clicking an interactable object triggers the interaction. Depending on the current training course step, the configured interaction will automatically completed. Therefore, if in your current step you have to grab a sphere _or_ touch a cube, clicking on the sphere will actually grab it and touching the cube will trigger "touching". As said before, this very simplified interaction system is a balance between functionality and usability with a weight on usability.
+
+Complex interaction such as [_snap object_](../innoactive-creator/default-conditions.md#snap-object) are divided into two interaction steps: clicking on the respective object to snap will pick the object up and place it into an **inventory**. A textual instruction now informs you to select the snap zone. By selecting the snap zone, the object is dropped or placed from within your inventory to the desired position. You can also interact with objects while they are inside your inventory, like using or touching it, by clicking the respective action buttons inside the inventory. 
+
+[![Inventory](../images/pro/03_inventory.png "Inventory and additional hints.")](../images/pro/03_inventory.png)
+
+### Switching between modes
+
+You can switch between VR-mode and desktop mode through the trainer menu on the bottom of your screen. Switching the mode will reload your current scene and switching is not possible while a training course is running.
+
+[![Switching modes](../images/pro/03-trainer-menu.png "Switching between VR and Desktop Mode.")](../images/pro/03-trainer-menu.png)
+
+## Advanced Mode
+
+An advanced desktop mode with more features and more possibilities to interact with your scene will come soon.
```

Comment:
I don't like that we are promising stuff "very soon". Cant we just say, it is planned or not mention it?

Reply:
it is not very soon, just soon :P but ill change it

Reply:
Maybe change it to "in a future release", that way makes sense if you read this in 2035 where you can just switch to a higher docu version.